containers/proxy-httpd-image/Dockerfile

```diff
@@ -39,7 +39,7 @@ RUN zypper --gpg-auto-import-keys --non-interactive install --auto-agree-with-li
 RUN zypper --gpg-auto-import-keys --non-interactive install --auto-agree-with-licenses \
  python3-rhnlib spacewalk-proxy-broker \
  spacewalk-proxy-common spacewalk-proxy-package-manager \
- spacewalk-proxy-redirect spacewalk-ssl-cert-check
+ spacewalk-proxy-redirect spacewalk-ssl-cert-check spacewalk-proxy-html
```

Comment:
Can you give an insight of why do we need to install this package as well?

Reply:
this package gives the nice HTML page for the proxy